lib/rules/font-family-generic-fallback/README.md

```diff
@@ -0,0 +1,39 @@
+# font-family-generic-fallback
+
+Require a generic fallback to any font-family declaration.
```

Comment:
Can we change this to the following, as this is [now a `no` rules](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#single-line-descriptions):

Disallow missing generic families in lists of font family names.